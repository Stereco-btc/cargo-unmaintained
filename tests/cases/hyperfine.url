https://github.com/sharkdp/hyperfine
