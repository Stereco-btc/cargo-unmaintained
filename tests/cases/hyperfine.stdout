rdrand (https://github.com/nagisa/rust_rdrand updated [..] days ago)
    rand_core (requirement: ^0.3, version used: 0.3.1, latest: 0.6.4)
atty (https://github.com/softprops/atty updated [..] days ago)
    hermit-abi (requirement: ^0.1.6, version used: 0.1.19, latest: 0.3.3)
cloudabi (https://github.com/nuxinl/cloudabi updated [..] days ago)
    bitflags (requirement: ^1.0, version used: 1.3.2, latest: 2.4.0)
statistical (https://github.com/JeffBelgum/statistical updated [..] days ago)
    num (requirement: ^0.2, version used: 0.2.1, latest: 0.4.1)
    rand (requirement: ^0.6, version used: 0.6.5, latest: 0.8.5)
